### DISCUSSION TITLE
First note(s) delayed when begenning to play

### DISCUSSION THREAD

**Discussion by supergedeon** [GrandOrgue/grandorgue#1852] [Help]
Hi everybody.
I notice a strange phenomenon: after starting GO and having loaded Friesach sample set, the first note (perhaps a few notes) shows a large latency. After those notes have been played, latency effect is not noticeable anymore. I noticed this when using a laptop under Windows 10, as well as with a desktop computer under Linux. 
As an information, in the Linux machine, buffer is set to 512 samples. Another information: in both cases, RAM is full up (the sample set size has been reduced to fit to the available memory). There is no antivirus protection and the computer is not connected to the Net. All it has to do is run GO. 
It's not a problem in itself, but I would be glad to have an explanation.

**Comment by oleg68** [comment]
What audio devises are selected in Settings->Audio under Windows and Linux?

**Reply by supergedeon** [reply]
I have two different setups in two different houses. The device with Windows is the DAC of a Simaudio Moon 340ix amplifier, and with Linux, it'an EMU 0404 audio/Midi interface.
I thank you for your reply, but I think the answer is in the post by midimusic below.

**Comment by midimusic** [comment]
I suspect the clue is in your statement that the memory is full up. I suspect that GO is partly contained in a swap file (swap files are slow). The first notes will inevitably require additional memory to be allocated and it will be coming via the swap file.

Solution: Try a smaller sample set and see whether the problem goes away.

csw900

**Reply by supergedeon** [reply]
Thank you for your answer;
The point is that, on the machine under Linux, the cache size is 15G+ with a 16G memory bank. So the whole sample set (cached by GO when first opened) seems to ly in the system's cache.
I'll try what you suggest with the demo sample set and I keep you informed.
Alain
